### DISCUSSION TITLE
KeyError (Note that I am using a custom theme for dark purple-blue)

### DISCUSSION THREAD

**Discussion by Muhammad-Yusr** [TomSchimansky/CustomTkinter#2061] [Q&A]
File "D:\Folders\Programming\Python Projects\UI-based\Task Manager\main.py", line 49, in <module>
    checkbox = ck.CTkCheckBox(entry_tab, text="Insert Mode", variable=insert_mode)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ADMIN\AppData\Roaming\Python\Python311\site-packages\customtkinter\windows\widgets\ctk_checkbox.py", line 54, in __init__
    self._fg_color = ThemeManager.theme["CTkCheckbox"]["fg_color"] if fg_color is None else self._check_color_type(fg_color)
                     ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
KeyError: 'CTkCheckbox'

Customtkinter setup:
![image](https://github.com/TomSchimansky/CustomTkinter/assets/63252676/8e60369e-7f49-4ef2-a2db-a70a40bba517)

JSON file checkbox part:
![image](https://github.com/TomSchimansky/CustomTkinter/assets/63252676/ac5c7fc4-281b-43f5-a229-faef21b0efe8)

and yes the path is correct it worked for other objects like buttons and labels

**Comment by Akascape** [comment]
@MoYusr Check the spellings properly, the b in box is in upper case.

Your Code:
```
ThemeManager.theme["CTkCheckbox"]
```
Correction:
```
ThemeManager.theme["CTkCheckBox"]
```
